src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCompiler.cs

```diff
@@ -4256,7 +4261,66 @@ private void GenerateOneCode()
                     //:         break Backward;
                     {
                         string str = _strings![Operand(0)];
+                        Label multiCode = DefineLabel();
+                        if (!IsRightToLeft())
+                        {
+                            // if (runtextend - runtextpos < c)
+                            Ldloc(_runtextendLocal!);
+                            Ldloc(_runtextposLocal!);
+                            Sub();
+                            Ldc(str.Length);
+                            BgeFar(multiCode);
+                            // if (!_caseInsensitive && _maxBacktrackPosition != -1 && runtextpos > _maxBacktrackPosition)
```

Comment:
Nit: consider removing the _ in these comments from _maxBacktrackPosition... it's actually a local in the compiled implementation rather than the field it is in the interpreter